core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -767,6 +801,21 @@ private[spark] class SparkSubmit extends Logging {
     (childArgs, childClasspath, sparkConf, childMainClass)
   }
 
+  private def renameResourcesToLocalFS(resources: String, localResources: String): String = {
+    if (resources != null && localResources != null) {
+      val localResourcesSeq = Utils.stringToSeq(localResources)
+      Utils.stringToSeq(resources).map { resource =>
```

Comment:
Use localResourcesSeq?

Reply:
@srowen the comment is resolved I made the changes required but I need `isKubernetesCluster` as well.